deep-learning/_posts/2017-03-30-fcnn.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: review
+title: "FCNN: Fully Convolutional Networks for Semantic Segmentation"
+tags: deep-learning CNN essentials segmentation
+cite:
+    authors: "J. Long, E. Shelhamer, T. Darrell"
+    title:   "Fully Convolutional Networks for Semantic Segmentation"
+    venue:   "proceedings of CVPR 2015, p.3431-3440"
+pdf:   "http://www.cv-foundation.org/openaccess/content_cvpr_2015/html/Long_Fully_Convolutional_Networks_2015_CVPR_paper.html"
+---
+
+ 
+One of the first and most successful CNN-based segmentation paper.  The proposed model is a simple sequence of convolution and maxpooling layers but with one (or more) up-sampling (or deconvolution) layers at the end.  The reasons for these up-sampling layers is to force the output to have the same size than the input and thus to have a segmentation map with the right size.
```

Comment:
I'll remove deconvolution, because `upsampling` and `deconvolution` is two different things.